### PR TITLE
test(cta-block): ensure sameHeight function runs within tabs-extended

### LINKED ISSUE #8901: [Test Scenario]: CTA Block Item Rows within Tabs Extended
**Engineering info:**  
- Innovation Team engineer: @andy-blum 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2479
- DPO consulting engineer: N/A

-----

### Background
#8820 revealed an issue where elements that leverage the `sameHeight` utility but are initially rendered in an element that has the `hidden` attribute are given a `0px` height. The specific scenario under which this occurred was with markup like the following:
```
<dds-tabs-extended>
  <dds-tab label="First tab">
    <dds-cta-block>
      <dds-cta-block-item-row>
        <dds-cta-block-item>
          ...
        </dds-cta-block-item>
      </dds-cta-block-item-row>
    </dds-cta-block>
  </dds-tab>
  ...
</dds-tabs-extended>
```

#8881 fixes the issue, but there is not currently a place to write tests for the scenario. However, this seems like the type of thing that warrants automated test coverage.

### Component and test scenario title

CTA Block – When rendered inside Tabs Extended, CTA Block Items within CTA Block Item Rows should have equal heights.

### Test scenario steps

1. Given a `DDSCTABlock` is placed within a `DDSTabsExtended` element
2. And a `DDSCTABlockItemRow` is placed within it,
3. And that row has one or multiple `DDSCTABlockItem` children
4. Then those children should have equal heights
5. And their heights should not equal `0px`.

### PR DESCRIPTION
**Pull request by andy-blum**
### Related Ticket(s)

Closes #8901 

### Description

Adds a CTA Block story with cta block items inside a tabbed interface and a corresponding test to ensure cta block items have their height values properly set on display

### Changelog

**Changed**

- Expanded cta block test coverage

### CODE REVIEW

File: packages/web-components/tests/e2e-storybook/cypress/integration/cta-block/cta-block.e2e.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+/* eslint-disable cypress/no-unnecessary-waiting */
+describe('dds-cta-block (desktop)', () => {
+  beforeEach(() => {
+    cy.visit(`/${_path}`);
+    cy.injectAxe();
+    cy.viewport(1280, 780);
+  });
+
+  xit('should check a11y', () => {
```
Comment (jkaeser) [inline packages/web-components/tests/e2e-storybook/cypress/integration/cta-block/cta-block.e2e.js:25]:
```suggestion
  it('should check a11y', () => {
```

File: packages/web-components/tests/e2e-storybook/cypress/integration/cta-block/cta-block.e2e.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        // Ensure each set of measurements are identical non-zero values
+        .each(heightSet => {
+          const isZero = heightSet.includes(0);
+          const isAllSame = heightSet.every(val => val === heightSet[0]);
+          expect(isZero).to.be.false;
+          expect(isAllSame).to.be.true;
+        });
+    }
+
+    cy.wait(forHeightsToBeSet)
```
Comment (jkaeser) [inline packages/web-components/tests/e2e-storybook/cypress/integration/cta-block/cta-block.e2e.js:70]: Your choice of variable name makes this so satisfying 🤌

**Review by jkaeser — CHANGES_REQUESTED** [review]
@andy-blum Looking great! Just a couple things to tidy up:
1. There's a single typo
2. Setting the "Content item type" knob to "button" doesn't render any CTAs (https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/8991/iframe.html?id=components-cta-block--within-tabs&knob-Heading%20(required)_CTABlock=Take%20the%20next%20step&knob-Content%20item%20type_WithinTabs=button&knob-Number%20of%20content%20items_WithinTabs=5)

File: packages/web-components/tests/e2e-storybook/cypress/integration/cta-block/cta-block.e2e.js (hunk 2 above)
Comment (andy-blum) [reply packages/web-components/tests/e2e-storybook/cypress/integration/cta-block/cta-block.e2e.js:70]: 🤌

**Comment by andy-blum** [comment]
@jkaeser thanks for the review!

To your second point:
> Setting the "Content item type" knob to "button" doesn't render any CTAs

This matches the implementation of the [With Content Items version](https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/8991/iframe.html?id=components-cta-block--with-content-items&knob-Heading%20(required)_CTABlock=Take%20the%20next%20step&knob-Content%20item%20type_WithContentItems=button&knob-Number%20of%20content%20items_WithContentItems=3)

**Review by jkaeser — APPROVED** [review]

**Review by IgnacioBecerra — APPROVED** [review]
LGTM!

**Pull request merged**